tests/integration-e2e/test_plugin.py

```diff
@@ -132,6 +133,8 @@ class ACellEngine(EngineBase):
     InnerConnectivityCls = ACellSynapseManager
 
 
+@pytest.mark.skip(
+    reason="Cannot test with SONATA configs, no SONATA parameter for Engine")
```

Comment:
@ferdonline  I disable this test because I cann't find a way to convert it to SONATA config.  Do you have any suggestion?

Reply:
You can leave it disabled for the moment. I will refactor this so that we can keep testing alternate engines even without Sonata syntax